File: RoomsApiCrudBackEnd/Entities/Room.cs
namespace RoomsApiCrudIdentity.Entities;

public class Room
{
    public int Id { get; set; } = default!;
    public string Name { get; set; } = default!;
    public int Capacity { get; set; } = default!;
    public int OfficeId { get; set; } = -1;
}

File: MainProject/acme-rooms/src/app/models/room.ts
import { numberAttribute } from "@angular/core";

export class Room {
    public id: number;
    public name: string;
    public capacity: number;
    public officeId: number;

    constructor () {
        this.id = -1;
        this.name = '';
        this.capacity = -1;
        this.officeId = -1;
    }
}
